### ISSUE TITLE
Data information&other tasks' performances

### ISSUE THREAD

**Issue by chengsetiankong** [jiesutd/PyTorchSeqLabel#4]
There are two questions I want to ask you:
1, The numbers of sentences of data used in your code are 14987, 3466, 3684 or 14041, 3250, 3453 (train, dev, test respectively). Can you tell me?
2, Whether your code can obtain comparable performances in CoNLL03 Germany NER data as [Lample .etc, NAACL16](http://www.aclweb.org/anthology/N/N16/N16-1030.pdf) and in WSJ data as [Ma .etc, ACL16](http://www.aclweb.org/anthology/P/P16/P16-1101.pdf). That is, how does your code perform in other tasks compared to existing related works? Did you have a try?

**Comment by jiesutd** [comment]
Hi @chengsetiankong , 
1. The data I use is 14987, 3466, 3684. Notice the result of NER is not stable(90.8~91.4), you may try different seed to check its volatility.
2. I tried my implementation of WSJ data, it roughly gives 97.46 on POS tagger without tuning the hyperparameter. Guessing it may give a better result if tuning the hyperparameter.

**Comment by chengsetiankong** [comment]
Get it. Thanks for your answer!
